src/BitmovinPlayer.test.tsx

```diff
@@ -142,6 +142,35 @@ describe('BitmovinPlayer', () => {
       });
     });
 
+    it('should initialize the default UI with the provided UI config', () => {
+      const { getBySelector } = render(
+        <BitmovinPlayer
+          config={{
+            ...playerConfig,
+            ui: {
+              metadata: {
+                title: 'Sintel',
+                description: 'A short film by Blender Foundation',
+              },
+            },
+          }}
+        />,
+        {
+          queries,
+        },
+      );
+
+      expect(getBySelector('.bmpui-ui-titlebar')).toBeInTheDocument();
+
+      expect(getBySelector('.bmpui-label-metadata-title')).toBeInTheDocument();
+      expect(getBySelector('.bmpui-label-metadata-title')).toHaveTextContent('Sintel');
+
+      expect(getBySelector('.bmpui-label-metadata-description')).toBeInTheDocument();
+      expect(getBySelector('.bmpui-label-metadata-description')).toHaveTextContent(
```

Comment:
All these expectations could be moved to a helper function to improve readability e.g. at the bottom of the file. But fine to keep it this way as well.

Reply:
Fair point, but I'd rather keep them as-is to make the test itself more readable (no navigation required) :)